plugins/modules/aci_l3out_floating_svi_path.py

```diff
@@ -0,0 +1,485 @@
+    access_encap = module.params.get("access_encap")
+
+    aci = ACIModule(module)
+
+    node_dn = "topology/pod-{0}/node-{1}".format(pod_id, node_id)
+
+    tDn = None
+    if domain_type == "physical":
+        tDn = "uni/phys-{0}".format(domain)
+    elif domain_type == "vmware":
+        tDn = "uni/vmmp-VMware/dom-{0}".format(domain)
+
+    aci.construct_url(
+        root_class=dict(
+            aci_class="fvTenant",
+            aci_rn="tn-{0}".format(tenant),
+            module_object=tenant,
+            target_filter={"name": tenant},
+        ),
+        subclass_1=dict(
+            aci_class="l3extOut",
+            aci_rn="out-{0}".format(l3out),
+            module_object=l3out,
+            target_filter={"name": l3out},
+        ),
+        subclass_2=dict(
+            aci_class="l3extLNodeP",
+            aci_rn="lnodep-{0}".format(node_profile),
+            module_object=node_profile,
+            target_filter={"name": node_profile},
+        ),
+        subclass_3=dict(
+            aci_class="l3extLIfP",
+            aci_rn="lifp-{0}".format(interface_profile),
+            module_object=interface_profile,
+            target_filter={"name": interface_profile},
+        ),
+        subclass_4=dict(
+            aci_class="l3extVirtualLIfP", aci_rn="vlifp-[{0}]-[{1}]".format(node_dn, encap), module_object=node_dn, target_filter={"nodeDn": node_dn}
+        ),
+        subclass_5=dict(
+            aci_class="l3extRsDynPathAtt",
+            aci_rn="rsdynPathAtt-[{0}]".format(tDn),
+            module_object=tDn,
+            target_filter={"tDn": tDn},
+        ),
+        child_classes=["l3extVirtualLIfPLagPolAtt"],
+    )
+
+    aci.get_existing()
+
+    if state == "present":
+        child_configs = []
+        if enhanced_lag_policy is not None and domain_type == "vmware":
+            existing_enhanced_lag_policy = ""
+            if isinstance(aci.existing, list) and len(aci.existing) > 0:
+                for child in aci.existing[0].get("l3extRsDynPathAtt", {}).get("children", {}):
+                    if child.get("l3extVirtualLIfPLagPolAtt"):
+                        existing_enhanced_lag_policy = (
+                            child.get("l3extVirtualLIfPLagPolAtt")
+                            .get("children")[0]
+                            .get("l3extRsVSwitchEnhancedLagPol")
+                            .get("attributes")
+                            .get("tDn")
+                            .split("enlacplagp-")[1]
+                        )
+                        if enhanced_lag_policy == "":
+                            child_configs.append(
+                                dict(
+                                    l3extVirtualLIfPLagPolAtt=dict(
+                                        attributes=dict(status="deleted"),
+                                    ),
+                                )
+                            )
+
+            if enhanced_lag_policy != "":
+                child = [
```

Comment:
should we rename this to children, since you will be appending multiple items to it in some scenarios

Reply:
Only one grand-child can be added. The other case involves deleting an existing grand-child and replacing it with another(update scenario). So I don't think the name is apt.